Samples/excel-add-in-custom-functions-using-web-workers/src/taskpane/taskpane.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+ * See LICENSE in the project root for license information.
+ */
+
+/* global console, document, Excel, Office */
+
+Office.onReady(() => {
+  document.getElementById("executeCFWithoutWebWorker").onclick = executeCFWithoutWebWorker;
+  document.getElementById("executeCFWithWebWorker").onclick = executeCFWithWebWorker;
+});
+
+
+// Execute Custom Function without WebWorker.
+async function executeCFWithoutWebWorker() {
+  try {
+    await Excel.run(async (context) => {
+      context.workbook.worksheets.getItemOrNullObject("Sample").delete();
+      const sheet = context.workbook.worksheets.add("Sample");
+
+      let range = sheet.getRange('A1');
+      range.values = [["=WebWorkerSample.TEST_UI_THREAD(20000)"]];
+      range.calculate();
+
+      sheet.activate();
+      await context.sync();
+    });
+  } catch (error) {
+    showStatus(`Exception when executeCFWithoutWebWorker: ${JSON.stringify(error)}.`, true);
+  }
+}
+
+// Execute Custom Function with WebWorker.
```

Comment:
Since this comment is equally as descriptive as the function name, go ahead and remove it.